client/me/account-close/main.jsx

```diff
@@ -46,10 +46,10 @@ class AccountSettingsClose extends Component {
 		showSiteDropdown: true,
 	};
 
-	UNSAFE_componentWillReceiveProps = ( nextProps ) => {
+	componentDidUpdate = () => {
```

Comment:
nit: it should be a classic method on a prototype.

Reply:
Ha, I wonder how it ended up being an arrow function in the first place. Fixing before 🚢 .